src/android/CameraLauncher.java

```diff
@@ -124,16 +127,9 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
             this.mediaType = PICTURE;
             this.mQuality = 80;
```

Comment:
I would get rid of all of these defaults here. I like your use of `optInt`, and passing in defaults; would suggest just eliminating the forced defaults above and go with the single line initialization in the method.

FWIW, here's how I ported this to my local copy:

        if (action.equals("takePicture")) {
            int srcType = args.optInt(2, CAMERA);
            int destType = args.optInt(1, FILE_URI);

            // Process the rest of the configuration arguments
            processConfiguration(args);
            ...